composer.json

```diff
@@ -1,12 +1,32 @@
 {
-  "name": "upstatement/timber-starter-theme",
+  "name": "timber/starter-theme",
```

Comment:
This would require us to re-add it to packagist.org, right?

Reply:
I think it does, I think that it would make sense though. And then @jarednova has to deprecate the old version from @upstatement to point to the new one.

Reply:
Yes, it makes sense :). I just wanted to make sure we keep track of this todo.